Gemfile.lock

```diff
@@ -197,7 +197,7 @@ GEM
       faraday (>= 1.0, < 3.a)
     google-cloud-errors (1.4.0)
     google-cloud-translate-v2 (0.5.0)
-      faraday (>= 0.17.3, < 3.a)
+      faraday (>= 0.17.3, < 2.a)
```

Comment:
suggestion: Manually updating the lock file isn't a great idea, and probably why the deployment failed. You could try to explicitly set the faraday gem version in the gemfile. All of our gems seem to support faraday v1 as a dependency, so you can try to use version 1.10.3.